slh-dsa/Cargo.toml

```diff
@@ -0,0 +1,42 @@
+[package]
+name = "slh-dsa"
+version = "0.0.1"
+edition = "2021"
+
+[dependencies]
+hybrid-array = {version = "0.2.0-rc.8", features = ["extra-sizes"]}
+typenum = {version = "1.17.0", features = ["const-generics"]}
```

Comment:
You can access this as `hybrid_array::typenum`, which also requires `typenum` v1.17, and also enables the `const-generics` feature

Reply:
I can fix this up after merging.